engines/griffon/dialogs.cpp

```diff
@@ -439,10 +450,20 @@ void GriffonEngine::configMenu() {
 						if (config.effects)
 							config.effects = false;
 						break;
-					case 13:
+					case 11:
+						if (!ConfMan.getBool("tts_enabled")) {
+							ConfMan.setBool("tts_enabled", true);
+						}
+						break;
+					case 12:
+						if (ConfMan.getBool("tts_enabled")) {
+							ConfMan.setBool("tts_enabled", false);
+						}
+						break;
+					case 16:
```

Comment:
I think this one is supposed to be 15, and the one below 16 (and not 17).
You only increased `MAXCURSEL` by 2, and its maximum value is 16, so `cursel` can never be 17. When you select `Exit + Save` it is 15, which does nothing as you have no case 15. And when you select `Exit`, it is 16, and it does a save and exit and not just exit.